data/academic_institution/aix.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Aix-Marseille University
+description: >
+  Aix-Marseille University is a public research university located in the region of Provence, southern France. 
+url: "https://www.univ-amu.fr/en"
+logo: academic_institution/aix.jpg
+continent: Europe
+courses:
+    - name: Functional Programming   
+location:
+     lat: 43.3048
+     long: 5.3774
+---
+s
```

Comment:
typo?

Reply:
Thanks for pointing this out. I'll check for typos if any.

> typo?